packages/html/src/events/form.rs

```diff
@@ -1,16 +1,24 @@
 use std::{any::Any, collections::HashMap, fmt::Debug};
 
 use dioxus_core::Event;
+use serde::{Deserialize, Serialize};
 
 pub type FormEvent = Event<FormData>;
 
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+#[serde(untagged)] // this will serialize Text(String) -> String and VecText(Vec<String>) to Vec<String>
+pub enum ValueType {
```

Comment:
Serde is an optional dependancy, this will fail if serde is disabled. We need to make serde always enabled to fix this

Reply:
I think that currently, this will enable serde as default for ValueType. Are you saying to include #[cfg(feature = "serialize")] for each enum's variants ?

Reply:
Did you mean something like this ?

```rust
#[cfg_attr(feature = "serialize", derive(Serialize, Deserialize))]
#[derive(Debug, Clone, PartialEq)]
#[serde(untagged)] // this will serialize Text(String) -> String and VecText(Vec<String>) to Vec<String>
pub enum ValueType {
    Text(String),
    VecText(Vec<String>),
}
```

Reply:
I meant removing `optional = true` in this line https://github.com/DioxusLabs/dioxus/blob/master/packages/html/Cargo.toml#L15. The serialize feature enables serialization of events for renderers, but we can enable deserialization of form values automatically.

Reply:
ohh, my bad i saw the wrong file, that's why I misunderstood your expectation.
I checked here by mistake: https://github.com/DioxusLabs/dioxus/blob/master/Cargo.toml

Ill change it.

Reply:
> I meant removing `optional = true` in this line https://github.com/DioxusLabs/dioxus/blob/master/packages/html/Cargo.toml#L15. The serialize feature enables serialization of events for renderers to reduce the binary size because of how many events Dioxus supports, but we can enable deserialization of form values automatically. I don't think it would effect the binary size very much

This change is causing issues, need to figure out why ... do you know what might be the cause?

Reply:
You also need to remove it from the serialize feature now that it is not optional